src/commands/support/reportedissues.ts

```diff
@@ -54,9 +89,28 @@ export const reportedissues: CommandDefinition = {
                 replyWithEmbed(msg, genericReportedIssuesEmbed);
                 return;
             }
-            if (args.length === 1 && args.at(0) === 'autopilot') {
-                replyWithEmbed(msg, autopilotEmbed);
-                return;
+
+            if (args.length === 1) {
+                if (args.at(0) === 'help') {
+                    msg.reply({ embeds: [helpEmbed] });
```

Comment:
```suggestion
                    replyWithEmbed(msg, helpEmbed);
```
Wouldn't this be better for consistency? Or is there a reason to do a regular reply? The only reason is in case the command is a reply to another message, but that is unlikely to be the case anyway for help/list. And even if it is, it might be the intended purpose?